CalculatorConsole/MathematicalOperations.cs

```diff
@@ -0,0 +1,247 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorConsole
+{
+    internal static class CalculatorOperations
```

Comment:
This class should not contain any Console Write/Read Operations, only mathematical operations, move the non mathematical operations in the CalcualtorLogic class, as per our discussion